nixos/modules/programs/alvr.nix

```diff
@@ -0,0 +1,33 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
+
+let
+  cfg = config.programs.alvr;
+in
+{
+  options = {
+    programs.alvr = {
+      enable = mkEnableOption (lib.mdDoc "ALVR");
```

Comment:
You should state in the enable option what ALVR is

Reply:
Done